cli/util.go

```diff
@@ -254,6 +254,14 @@ func splitString(s string) []string {
 	})
 }
 
+// Split the string into a command and its arguments.
+func splitCommand(s string) (string, []string) {
+	cmdAndArgs := strings.Fields(s)
```

Comment:
Better use `shlex.Split()` this is shell aware

Reply:
Updated to use `shlex.Split()`: https://github.com/nats-io/natscli/pull/1174/commits/48fbe2963604a9af4810410e04a683c03a93b490.

@ripienaar do you think it's better to return `(string, []string, error)` or return a result struct + error?